spec/factories/nonprofits.rb

```diff
@@ -18,9 +18,9 @@
       }
     end
 
-    after(:create) {|nonprofit, evaluator|
-      create(:supporter, nonprofit: nonprofit)
-    }
+    # after(:create) {|nonprofit, evaluator|
+    #   create(:supporter, nonprofit: nonprofit)
+    # }
```

Comment:
Might as well just delete it.

Reply:
Oops, agreed, I commented it out to test and then forgot to delete it.